src/main/java/com/google/sps/servlets/NewGameServlet.java

```diff
@@ -14,32 +14,105 @@
 
 package com.google.sps.servlets;
 
+
+import com.google.api.core.ApiFuture;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.FirestoreOptions;
+import com.google.cloud.firestore.WriteResult;
+import com.google.sps.daos.CounterDao;
 import java.io.IOException;
-import com.google.common.collect.ImmutableList; 
-import java.util.List;
+import java.util.HashMap;
+import java.util.Map;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/new-game")
 public class NewGameServlet extends HttpServlet {
 
+  // Reference to Firestore database
+  private Firestore firestoreDb;
```

Comment:
Can you use the firestore reference in the context that Armando created instead of creating another one here?

Reply:
This will be resolved on the next PR since I'm creating more classes including the GameDAO